src/app/pages/SocialAuth/socialauth.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component } from '@angular/core';
+import { LoginService } from 'src/app/services/SocialAuth/socialauth.service';
+import {
+  AuthService,
+  FacebookLoginProvider,
+  GoogleLoginProvider,
+  SocialUser
+} from 'angularx-social-login';
+import { Router } from '@angular/router';
+import { ToastrService } from 'ngx-toastr';
+import { NgxSpinnerService } from 'ngx-spinner';
+
+@Component({
+  selector: 'app-social-login',
+  templateUrl: './socialauth.component.html',
+  styleUrls: ['./socialauth.component.scss']
+})
+export class SocialLoginComponentt {
+  user: SocialUser = null;
+  loggedIn: boolean;
+  loading: boolean = true;
+
+  // inject services and depe
```

Comment:
This comment is not really helpful, could you please do away with it or make it more helpfule?